save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt

```diff
@@ -329,21 +325,68 @@ internal class OrganizationController(
         .switchIfEmptyToNotFound {
             "Could not find an organization with name $organizationName."
         }
+        .filter {
+            it.status == OrganizationStatus.CREATED
+        }
+        .switchIfEmptyToNotFound {
+            "The organization has the status not ${OrganizationStatus.CREATED}"
+        }
         .filter {
             organizationPermissionEvaluator.hasPermission(authentication, it, Permission.DELETE)
         }
         .switchIfEmptyToResponseException(HttpStatus.FORBIDDEN) {
             "Not enough permission for deletion of organization $organizationName."
         }
         .filter {
-            organizationService.organizationHasNoProjects(it.name)
+            organizationService.organizationHasNoProjects(it.name) || OrganizationStatus.valueOf(status.uppercase()) == OrganizationStatus.BANNED
         }
         .switchIfEmptyToResponseException(HttpStatus.CONFLICT) {
             "There are projects connected to $organizationName. Please delete all of them and try again."
         }
```

Comment:
No. If a person deletes an organization, he needs to delete all its projects. If someone bans an organization, he does not need to delete all projects from it or assign the Banned status to them. This will make the program.

Reply:
Oh sure